submissions/satisfactorio/satisfactorio.md

```diff
@@ -0,0 +1,91 @@
+
+### How?
+
+As a first step, I set about figuring out how to connect belts directly to machines. I tried a few things with Loaders, but quickly found that they have a strong preference for one belt lane over the other. If the input isn't saturated, then only the right lane gets used. If that lane fills, then the other gets started, and once the bottleneck clears you end up with doubled throughput for a while, which wasn't ideal.
+
+[[Video: one-lane-loaders.mp4 -- The miner only works on one belt lane, not both.]]
+
+I looked into other mods such as [Miniloader](https://mods.factorio.com/mod/miniloader) and [Editor Extensions](https://mods.factorio.com/mod/EditorExtensions) to see how they handled loading items onto both belt lanes, and discovered that the solution was to have a pair of invisible inserters, one outputting to each lane.
+
+Before too long I had a miner putting items on a belt, the belt could go into a smelter to produce ingots, then those go into a constructor to make plates, which get put in a storage container. Cool!
+
+[[Video: tiny-production-line.mp4 -- Mining iron, smelting ingots, and constructing plates]]
+
+I got a taste for modding and before I knew it I decided that whereas the mods that inspired me were limited in scope (one Satisfactory mod just adds Inserters, another adds Factorio recipes), *my* scope would be... *everything*. I was going to port everything Satisfactory has to offer, so I started writing down the feature list I would need to work on.
+
+Turns out, there's a lot of features! So I got to work on the easier ones. Recipes were of course quite easy to bring over, but others... not so much!
+
+While it's not possible to create entirely new types of building in Factorio, the desired effects can be reached through the use of "compound entities". That is, multiple different buildings stacked on (or at least near) each other. For example, the Miner doesn't actually mine items onto the belt; that would again result in only one lane being used. Instead, it places its results into an invisible chest, and the chest is then unloaded onto the belt by a Miniloader-like structure. Through this method, all kinds of new systems are possible!
+
+### Major changes
+
+While fundamentally Factorio and Satisfactory both share the similar core of "build stuff to build more stuff", there are some major differences that I had to make work.
+
+First of all, there is no hand-crafting in Satisfactory. At least, not in the sense that you can just fill up a crafting queue and go about your business until it's done. Instead, you have to go to the Craft Bench and you can't do anything else while working there. To replicate this, I made an assembling machine that doesn't require power, but only works if the player has the machine open on screen.
```

Comment:
Maybe add a link here to the Craft Bench too, for the players who hasn't played Satisfactory.

https://satisfactory.fandom.com/wiki/Craft_Bench